x-pack/plugins/cases/public/components/use_push_to_service/index.tsx

```diff
@@ -114,79 +113,24 @@ export const usePushToService = ({
     }
 
     return errors;
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [actionLicense, caseStatus, connectors.length, connector, loadingLicense, permissions.update]);
-
-  const pushToServiceButton = useMemo(
```

Comment:
`usePushToService` does not return any more components. I created separate components in separate files.